front/src/index.ts

```diff
@@ -1,15 +1,16 @@
 import 'phaser';
 import GameConfig = Phaser.Types.Core.GameConfig;
-import {GameScene} from "./GameScene";
-import {Connexion} from "./Connexion";
+import {GameManager} from "./Phaser/Game/GameManager";
 import {RESOLUTION} from "./Enum/EnvironmentVariable";
 
+let gameManager = new GameManager();
+
 const config: GameConfig = {
     title: "Office game",
     width: window.innerWidth / RESOLUTION,
     height: window.innerHeight / RESOLUTION,
     parent: "game",
-    scene: [GameScene],
+    scene: gameManager.GameScenes,
```

Comment:
If we have many rooms and we want to switch room in-game, we can probably do this dynamically.
Here, we should probably have only one scene (as before), or 2-3 scenes (if we want to put a splash screen or a configuration page to edit controls (not everybody uses a AZERTY keyboard!))

Reply:
Yes and in GameManager, we can add new GameScene to manage other scene or menu.